### PR TITLE
Fork solhint-config-keep

### PR DESCRIPTION
**Pull request by nkuba**
Fork the configuration defined by Keep Network team and use it in Thesis projects.

### CODE REVIEW

File: README.md
Comment (dimpar) [inline README.md:1]: I think we can get rid of this info: `Solhint is an alternative to Solium(ethlint) which is no longer maintained.` At this point nobody probably cares.

**Comment by dimpar** [comment]
Just one small comment. Other things lgtm.

**Review by r-czajkowski — APPROVED** [review]
LGTM

File: README.md
Comment (nkuba) [reply README.md:1]: b621aeb360536e11cb097f20efa04a81acd3db09

**Pull request merged**